addon/components/polaris-popover/content.js

```diff
@@ -82,7 +82,7 @@ export default Component.extend({
       return;
     }
     const paneContent = pane.firstElementChild;
-    const paneContentRect = paneContent.getBoundingClientRect();
+    const paneContentRect = getRectForNode(paneContent);
     Ember$('div.Polaris-Popover__Content', content).css({
```

Comment:
may use the native api instead of jQuery?

Reply:
I think that's beyond the scope of this PR since this was just a quick swap of an old util for a new one... we should look into removing JQuery but have hit difficulties doing so before.

Reply:
Sounds good